Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -426,12 +519,6 @@ SWIFT_CLASS_NAMED("EntitlementInfo")
 /// or shared to them by a family member. This can be useful for onboarding users who have had
 /// an entitlement shared with them, but might not be entirely aware of the benefits they now have.
 @property (nonatomic, readonly) enum RCPurchaseOwnershipType ownershipType;
-/// The underlying data for this <code>EntitlementInfo</code>.
-/// note:
-/// the content and format of this data isn’t documented and is subject to change,
-/// it’s only meant for debugging purposes or for getting access to future data
-/// without updating the SDK.
-@property (nonatomic, readonly, copy) NSDictionary<NSString *, id> * _Nonnull rawData;
```

Comment:
Hmm hold on, why is this gone? 🤨

Reply:
Ditto here:
```swift

 extension EntitlementInfo: RawDataContainer {
     // Docs inherited from protocol
     // swiftlint:disable missing_docs
     public var rawData: [String: Any] {
         return self.entitlement.rawData
     }
 }
```

And I can correctly use it from outside. This was changed in #1565.

I'd say it's not a ship stopper, just some implementation detail of how this is generated.